src/web/mod.rs

```diff
@@ -111,7 +111,7 @@ impl CratesfyiHandler {
 
         let shared_resources = Self::chain(&pool_factory, rustdoc::SharedResourceHandler);
         let router_chain = Self::chain(&pool_factory, routes.iron_router());
-        let prefix = PathBuf::from(env::var("CRATESFYI_PREFIX").unwrap()).join("public_html");
+        let prefix = PathBuf::from(env::var("CRATESFYI_PREFIX").expect("CRATESFYI_PREFIX environment variable does not exists")).join("public_html");
```

Comment:
What do you think, is this overkill?

```suggestion
        let prefix = PathBuf::from(env::var("CRATESFYI_PREFIX")
            .expect("the CRATESFYI_PREFIX environment variable is not set, \
                     see https://github.com/rust-lang/docs.rs/wiki/Developing-without-docker-compose \
                     for instructions on developing locally")
        ).join("public_html");
```

Reply:
The issue I feel with putting the link on the expect is that it is weird for this one to have it, and not the other ones, but if all of them have it, it gets pretty ugly, so maybe it needs to just check all the environment variables at the start, and print an error message, but I don't think it should be put into this pull request.

Reply:
```suggestion
        let prefix = PathBuf::from(env::var("CRATESFYI_PREFIX").expect("the CRATESFYI_PREFIX environment variable is not set")).join("public_html");
```